docs/examples/general/data_loading/external_input.ipynb

```diff
@@ -4,10 +4,10 @@
    "cell_type": "markdown",
```

Comment:
<p>This remarks has a broader application:</p><p>"on the CPU" sounds misleading - like it was bound to a particular CPU (which is not true, because, unlike GPU memory, host memory is equally accessible to all CPUs).</p><p>Perhaps we should switch from "on CPU" to "in host memory"?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Done</p>